dired-narrow.el

```diff
@@ -174,7 +174,8 @@ when `dired-narrow-exit-when-one-left' and `dired-narrow-enable-blinking' are tr
 (defun dired-narrow--restore ()
   "Restore the invisible files of the current buffer."
   (let ((inhibit-read-only t))
-    (remove-text-properties (point-min) (point-max) '(invisible))
+    (remove-text-properties (point-min) (point-max) '(invisible nil
```

Comment:
I'm not sure what the `nil`s are doing here?  From the docs of the function

```
The third argument PROPERTIES is a property list
whose property names specify the properties to remove.
(The values stored in PROPERTIES are ignored.)
```

I'm on Emacs 25 still, maybe the API was extended in newer Emacs?  Or is this simply a mistake?

Reply:
Note "property list" not "list of property names".  The [info docs][] actually say (emphasis added):

>The argument props specifies which properties to delete. It should have the form of a property list (see Property Lists): **a list whose elements are property names alternating with corresponding values**. **But only the names matter**—the values that accompany them are ignored. For example, here's how to remove the face property.
>
>     (remove-text-properties start end '(face nil))

[info docs]: https://www.gnu.org/software/emacs/manual/html_node/elisp/Changing-Properties.html

However `remove-text-properties` iterates over that plist, it must not particularly care when you pass it a list with only a single property name and no value.  However, when I added a second property to be deleted, I assumed I would need to follow the format of a plist.

No matter, though: while looking up the documentation here I found `remove-list-of-text-properties`, which just wants a simple list of properties to remove.  I changed my commit to use that instead of `remove-text-properties`.